plugin/src/node-builder.ts

```diff
@@ -123,19 +127,19 @@ export function nodeBuilder(
 
       const processor = processorMap[remoteType] || (() => Promise.resolve());
 
-      attachParentId(result, gatsbyApi);
+      attachParentId(result, gatsbyApi, pluginOptions);
 
       const node = {
         ...result,
         shopifyId: result.id,
-        id: gatsbyApi.createNodeId(result.id),
+        id: gatsbyApi.createNodeId(`${pluginOptions.typePrefix}_${result.id}`),
```

Comment:
We still have an underscore in this instance

Reply:
Oh geez... Fixed!